data/tests.json

```diff
@@ -729,5 +729,52 @@
                 "direction": "forward"
             }
         ]
+    },
+    {
+        "skip_rust": true,
+        "description": "test for `busway:left=lane`",
+        "tags": {
+            "busway:left": "lane",
+            "highway": "secondary",
+            "lanes": "2",
+            "oneway": "yes"
+        },
+        "driving_side": "right",
+        "output": [
+            {
+                "type": "bus_lane",
+                "direction": "backward"
```

Comment:
Is this right?

If I compare this to the first example in this file, it means
`cycleway:left=lane` has an opposite direction than `busway:left=lane`

Reply:
Please see my answer to @dabreegster.